woocommerce-google-analytics-integration.php

```diff
@@ -204,6 +204,84 @@ public function maybe_show_ga_pro_notices() {
 			WC_Admin_Notices::add_custom_notice( 'woocommerce_google_analytics_pro_notice', $notice_html );
 			update_option( 'woocommerce_google_analytics_pro_notice_shown', true );
 		}
+
+		/**
+		 * Get the path to something in the plugin dir.
+		 *
+		 * @param string $end End of the path.
+		 * @return string
+		 */
+		public function path( $end = '' ) {
+			return untrailingslashit( dirname( __FILE__ ) ) . $end;
+		}
+
+		/**
+		 * Get the URL to something in the plugin dir.
+		 *
+		 * @param string $end End of the URL.
+		 *
+		 * @return string
+		 */
+		public function url( $end = '' ) {
+			return untrailingslashit( plugin_dir_url( plugin_basename( __FILE__ ) ) ) . $end;
+		}
+
+		/**
+		 * Get the URL to something in the plugin admin assets dir.
+		 *
+		 * @param string $end End of the URL.
+		 *
+		 * @return string
+		 */
+		public function get_js_asset_url( $end = '' ) {
+			return $this->url( '/assets/js/build/' . $end );
+		}
+
+		/**
+		 * Get the path to something in the plugin admin dir.
```

Comment:
Similar to the above. Would be better to indicate it's the path to the build directory of js assets?

Reply:
Tweaked here https://github.com/woocommerce/woocommerce-google-analytics-integration/pull/261/commits/f2689e15d5ccc1a5637a1f7d026346722c5dc4a1